### PR TITLE
Heesoo song/master

### PR DESCRIPTION
**Pull request by MichielStock**

### CODE REVIEW

**Comment by MichielStock** [comment]
image
I think this might be a good start!

**Comment by jstaut** [comment]
Review suggestions (Jasper Staut)

Very nice project and notebook!

README
Why would gradient-free be better and what would be the arguments for choosing one or the other approach (pro's and con's of updating one at a time?)

reproduce "the" RSO function with "a" [...] to "the" back propagation method

0 introduction
use "\leq" instead of "<="

reproduce "the" RSO function [...] accuracy to "the classical back propagation method". [...] One convolutional "layer"

1-1 Parameters
A weight in wid -> $w_{i_d}$

2-3 Result
Maybe more explicitly state that you ran those models and that it is not a table from the paper

**Comment by wardvanbelle** [comment]
Review suggestions (Ward Van Belle)

Really nice notebook! 
* It is not necessary to first install the MLDataset package before running the notebook. When using Pluto notebooks, everything is installed automatically :)
* Maybe add some example images from the MNIST dataset? Always nice to see something visuals!
* I guess that the calculation times in sections 2-1 and 2-2 are based on a GPU? When I ran it on my CPU, it took a bit longer (around 5min I think?).
* In 2-3 you only show the comparison of the accuracy. Maybe also show a comparison of the loss since in machine learning it always is important to look at the loss and accuracy. This will also show the reader that the model is not overfitting on the training data :)
